requirements_all.txt

```diff
@@ -298,7 +298,7 @@ holidays==0.8.1
 # homeassistant.components.camera.onvif
 http://github.com/tgaugry/suds-passworddigest-py3/archive/86fc50e39b4d2b8997481967d6a7fe1c57118999.zip#suds-passworddigest-py3==0.1.2a
 
-# homeassistant.components.xiaomi
+# homeassistant.components.xiaomi_aqara
 https://github.com/Danielhiversen/PyXiaomiGateway/archive/0.3.2.zip#PyXiaomiGateway==0.3.2
```

Comment:
(for another PR)

@Danielhiversen could you push this to PyPi ? We are trying to move away from GitHub deps.

Reply:
will do